### DISCUSSION TITLE
[request] - Player command to transfer claim blocks

### DISCUSSION THREAD

**Discussion by ShaneBeee** [GriefPrevention/GriefPrevention#109] [Ideas]
SUGGESTION:

Let's say that two players or more want to join forces and build a bigger base together. The problem is each player has say 100 blocks. So one player joins another players "village" .... the player who joins will keep accruing usable blocks but never actually use them.
Could you add a command where a player could give another player their blocks?
This would allow several people to give their accrued blocks to the owner of the "village" allowing them to add them to the village and keep expanding it.

I think this would be super helpful in a lot of cases.
Thanks for reading

-Shane

**Comment by morelaid** [comment]
What's the status here?
Is this now possible after almost 2 years or is it still open?

**Comment by RoboMWM** [comment]
@morelaid this is an addon suggestion, so anyone who wishes to implement this in an addon can do so and link it here.

**Comment by RoboMWM** [comment]
> Is this now possible 

Yes, this has been possible via the available API even before this addon suggestion was made...
